rewrite.yml

```diff
@@ -50,7 +50,6 @@ recipeList:
   - org.openrewrite.java.recipes.NoMutableStaticFieldsInRecipes
   - org.openrewrite.java.recipes.RecipeEqualsAndHashCodeCallSuper
   - org.openrewrite.java.recipes.UseTreeRandomId
-  - org.openrewrite.java.recipes.migrate.RemoveTraitsUsageRecipes
   - org.openrewrite.staticanalysis.NeedBraces
```

Comment:
```suggestion
  - org.openrewrite.java.InlineMethodCalls
  - org.openrewrite.staticanalysis.NeedBraces
```

Reply:
This recipe replaces the older `RemoveTraitsUsageRecipes`, as indicated hre:
- https://github.com/checkstyle/checkstyle-openrewrite-recipes/issues/101#issuecomment-3241216969